lib/perl/Genome/Utility/Test.pm

```diff
@@ -312,10 +323,27 @@ sub data_dir_ok {
     return $data_dir;
 }
 
+sub get_test_dir {
```

Comment:
It seems like things using this could have used `data_dir_ok` immediately preceding it.